src/edu/csus/ecs/pc2/core/model/FinalizeData.java

```diff
@@ -133,4 +141,62 @@ public void setUseWFGroupRanking(boolean useWFGroupRanking) {
     public boolean isUseWFGroupRanking() {
         return useWFGroupRanking;
     }
+
+    /**
+     * Set whether to customize the minimum number of problems solved for each Honors field
+     *
+     * @param customizeHonorsSolvedCount
+     */
+    public void setCustomizeHonorsSolvedCount(boolean customizeHonorsSolvedCount) {
+        this.customizeHonorsSolvedCount = customizeHonorsSolvedCount;
+    }
+
+    /**
+     * Return whether or not minimum problems solved for honors fields is to be customized or not
+     *
+     * @return true if WF group rankings are to be used
```

Comment:
Javadoc for @return is wrong.  Looks like a copy/paste error.

Reply:
Updated!